wrappers/aws/aws_test.go

```diff
@@ -1554,3 +1570,128 @@ func TestNonTracedService(t *testing.T) {
 
 	tp.Reset()
 }
+
+func TestSESSendEmailNotMasked(t *testing.T) {
+	config.MaskSESMail = false
+	// Initilize trace plugin to set GlobalTracer of opentracing
+	tp := trace.New()
+
+	// Create a session and wrap it
+	sess := getSessionWithSESSendEmailResponse()
+	sesc := ses.New(sess)
+
+	input := &ses.SendEmailInput{
+		Source: aws.String("demo@thundra.io"),
+		Destination: &ses.Destination{
+			ToAddresses: []*string{aws.String("test@thundra.io")},
+		},
+		Message: &ses.Message{
+			Subject: &ses.Content{
+				Data: aws.String("subject-test"),
+				Charset: aws.String("UTF-8"),
+			},
+			Body: &ses.Body{
+				Html: &ses.Content{
+					Data: aws.String("html-test"),
+					Charset: aws.String("UTF-8"),
+				},
+				Text: &ses.Content{
+					Data: aws.String("test"),
+					Charset: aws.String("UTF-8"),
+				},
+			},
+		},
+	}
+	sesc.SendEmail(input)
+
+	// Get the span created for dynamo call
```

Comment:
Not important but we can change `dynamo`s in test comments to `ses`, same for the other SES test too.